README.md

```diff
@@ -1941,6 +1942,16 @@ Usage:
 
 ```
 
+### Pretender
+The addon uses [Pretender](https://github.com/pretenderjs/pretender) to mock the requests. It exposes the functions `getPretender` and `setPretender` to respectively get the Pretender server for the current test or set it. For instance,
+
+```javascript
+import { getPretender } from 'ember-data-factory-guy';
+
+// Passthrough 'data:' requests.
+getPretender().get('data:*', getPretender().passthrough);
```

Comment:
Thanks for the PR! :)  
Did you forget to use setPretender ?  or maybe can you add an example for that too ?

Reply:
I didn't have a use case myself for `setPretender` but still wanted to have it mentioned somewhere on the README.